cmd/chainbridge/account.go

```diff
@@ -23,53 +23,27 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
-// handleAccountsCmd manages the flags for the account subcommand
-// first, if the generate flag is set, if so, it generates a new keypair
-// then, if the import flag is set, if so, it imports a keypair
-// finally, if the list flag is set, it lists all the keys in the keystore
-func handleAccountsCmd(ctx *cli.Context) error {
-	err := startLogger(ctx)
-	if err != nil {
-		return err
-	}
-
-	// key directory is datadir/keystore/
-	datadir, err := getDataDir(ctx)
-	if err != nil {
-		return fmt.Errorf("failed to access the datadir: %s", err)
-	}
+// wrapHandler takes in a Cmd function (all declared below) and wraps
+// it in the correct signature for the Cli Commands
+func wrapHandler(hdl func(*cli.Context, string) error) cli.ActionFunc {
```

Comment:
handler should probably consume a struct instead of `string`. The reason for this is in the future i can assume there might be more parameters outside of `datadir` that other commands may need later on

Reply:
That's a good point, added.